### PR TITLE
Very minor polish

### PR DESCRIPTION
**Pull request by pablodanswer**
## Description
[Provide a brief description of the changes in this PR]

## How Has This Been Tested?
[Describe the tests you ran to verify your changes]

## Accepted Risk
[Any know risks or failure modes to point out to reviewers]

## Related Issue(s)
[If applicable, link to the issue(s) this PR addresses]

## Checklist:
- [ ] All of the automated tests pass
- [ ] All PR comments are addressed and marked resolved
- [ ] If there are migrations, they have been rebased to latest main
- [ ] If there are new dependencies, they are added to the requirements
- [ ] If there are new environment variables, they are added to all of the deployment methods
- [ ] If there are new APIs that don't require auth, they are added to PUBLIC_ENDPOINT_SPECS
- [ ] Docker images build and basic functionalities work
- [ ] Author has done a final read through of the PR right before merge

### CODE REVIEW

File: web/src/app/chat/lib.tsx [hunk 1]
```diff
@@ -698,7 +698,8 @@ export async function useScrollonStream({
       ) {
         // catch up if necessary!
         const scrollAmount = scrollDist.current + 10000;
-        if (scrollDist.current > 140) {
+        if (scrollDist.current > 300) {
```
Comment (rkuo-danswer) [inline web/src/app/chat/lib.tsx:701]: Would be nice to refactor out these magic numbers.

File: backend/danswer/db/search_settings.py [hunk 2]
```diff
@@ -183,7 +183,7 @@ def update_current_search_settings(
 
     # Whenever we update the current search settings, we should ensure that the local reranking model is warmed up.
     if (
-        current_settings.provider_type is None
+        search_settings.rerank_provider_type is None
```
Comment (rkuo-danswer) [inline backend/danswer/db/search_settings.py:186]: For my own edification, why does this check for rerank_provider_type?

**Review by rkuo-danswer — APPROVED** [review]

File: backend/danswer/db/search_settings.py (hunk 2 above)
Comment (pablonyx) [reply backend/danswer/db/search_settings.py:186]: Our most generalizable method of checking for cloud reranking models (which don't require a warm up).

File: web/src/app/chat/lib.tsx (hunk 1 above)
Comment (pablonyx) [reply web/src/app/chat/lib.tsx:701]: Agreed!

**Pull request merged**